.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
Pojawiły ci się niepotrzebne pliki .idea. Usuń je proszę tak jak było to opisane w jednym z poprzednich zadań